addon/index.js

```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 
 Ember.deprecate("ember-getowner-polyfill is now a true polyfill. Use Ember.getOwner directly instead of importing from ember-getowner-polyfill", false, {
-  id: "ember-getowner-polyfill.import"
+  id: "ember-getowner-polyfill.import",
+  until: '3.0.0'
```

Comment:
I think it should be `until: '2.0.0'` (since I plan to remove the whole module in ember-getowner-polyfill@2.0.0).